src/modules/AuthServer.pm

```diff
@@ -1582,6 +1582,25 @@ sub WriteServiceSettings {
     return 1;
 }
 
+# If the system is openSUSE Tumbleweed, return '/usr/lib64/openldap'.
+# Otherwise, return an empty string.
+sub GetLoadModulePath {
+    # In early January 2016, OpenLDAP on Tumbleweed was updated to no longer bunble
+    # database drivers in the slapd binary itself.
+    # Hence, the database drivers (bdb/mdb/hdb) must be explicitly loaded for Tumbleweed users.
+    open(my $osr, '<', '/etc/os-release');
+    while (my $row = <$osr>)
+    {
+        if ($row =~ /Tumbleweed/)
+        {
+            return '/usr/lib64/openldap';
```

Comment:
Hardcoding `lib64` is wrong, we still release TW even for i586...

You need to handle `lib` vs. `lib64` better.